fvtest/compilertriltest/ShiftAndRotateTest.cpp

```diff
@@ -1188,7 +1226,9 @@ TEST_P(UInt64MaskThenShift, UsingLoadParam) {
 
 INSTANTIATE_TEST_CASE_P(ShiftAndRotateTest, UInt64MaskThenShift, ::testing::Combine(
     ::testing::ValuesIn(static_cast< std::vector<std::tuple<std::tuple<uint64_t, uint64_t>, int32_t>> (*) (void)>(test_mask_then_shift_values)()),
-    ::testing::Values(std::make_tuple<const char*, uint64_t (*) (uint64_t, uint64_t, int32_t)>("lushr", static_cast<uint64_t (*) (uint64_t, uint64_t, int32_t)>(mask_then_shift_right))
+    ::testing::Values(
+        std::make_tuple<const char*, uint64_t (*) (uint64_t, uint64_t, int32_t)>("lushr", static_cast<uint64_t (*) (uint64_t, uint64_t, int32_t)>(mask_then_shift_right)),
+        std::make_tuple<const char*, uint64_t (*) (uint64_t, uint64_t, int32_t)>("lushl", static_cast<uint64_t (*) (uint64_t, uint64_t, int32_t)>(mask_then_shift_left))
```

Comment:
I don't really think we should be testing `lushl`. That opcode is being deprecated since it's functionally equivalent to its signed counterpart (see #2657).

Reply:
Right, I only added the lushl test because I've added support for a `lushl`+`land` combination. @fjeremic Should we still be testing for `lushl`?

Reply:
Left shifts have no bearing on the sign bit, it's the signedness of the right shift we have to be careful about. Yes we can remove the unsigned left shift tests.